content/automate/ManageIQ/AutomationManagement/AutomationManager/Lifecycle.class/provisioning.yaml

```diff
@@ -0,0 +1,14 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name:
+    name: Provisioning
+    inherits:
+    description:
+  fields:
+  - Relationship5:
+      value: "/AutomationManagement/AutomationManager/Provisioning/Profile/${/#user.normalized_ldap_group}#get_state_machine"
+  - Relationship6:
+      value: "/AutomationManagement/AutomationManager/Provisioning/StateMachines/${/#state_machine}/default"
```

Comment:
Note, we can remove a lot of indirection here if we don't care about getting a per ldap group state machine... we could go directly to:

```
"/AutomationManagement/AutomationManager/Provisioning/StateMachines/Provision/default"
```

Reply:
We can add it later if needed.